src/config/PipelinesConfig.json

```diff
@@ -24,6 +24,6 @@
     "name": "ont",
     "title": "ONT",
     "description": "A LIMS pipeline to support tracking for ONT Long-Read Sequencing",
-    "routes": ["samples"]
+    "routes": ["samples", "pools", "pool/new"]
```

Comment:
Do you think we need pools and pool new. We could have new pool on pools page. Not sure ??

Reply:
Not sure. Could add a clickable button that takes the user to the new pool page from the pools page? Not sure if thats an improvement or not though